docs/user-guide/advanced/Pandas_API.ipynb

```diff
@@ -353,6 +353,32 @@
     "tab.size"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "4023811f-0f23-4e4b-919c-c055b865d9d2",
+   "metadata": {},
+   "source": [
+    "### Table.values\n",
+    "Return a matricial representation of the DataFrame.\n",
+    "\n",
+    "Only the values in the DataFrame will be returned, the axes labels will be removed.\n",
+    "\n",
+    "**Returns:**\n",
```

Comment:
The "return" section in other functions that are of the `property` type is not present, as it can be in `columns` or `dtype`, so consider whether it is necessary to include it or if it adds nothing.

Reply:
I have tried to follow the actual Pandas documentation. That is the reason for the 'Returns' section. As can be seen here (https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.size.html), the 'size' method does not have a 'Returns' section, but the 'values' method does (https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.values.html).